balsamiq.md

```diff
@@ -0,0 +1,131 @@
+
+It took some effort to get it set up, but now we don't even think about it. It's really easy for the content writers, and we'll be able to extend it when we add yet more versions in the future.
+
+
+## Challenge #2: Adding play/pause to animated GIFs
+
+We noticed a while ago that our support team was getting great feedback when they would send animated GIFs to customers to explain solutions to problems they were having. The classic case of "show, don't tell." We realized that adding them to our documentation could be helpful too, and we added a few of them in the first version of our docs.
+
+But it can be distracting to read an article when there's an endlessly-looping animation in your line of sight. But if you only play it once, readers could miss it.
+
+I came across several sites that gave users an option to play the animated GIF when they clicked on it and I thought that would be great for our docs.
+
+I found a great jquery plugin that I wanted to use called [**gifplayer**](http://rubentd.com/gifplayer/).
+
+[![You know you want to click](https://media.balsamiq.com/images/docslikecode/gifplayer.png)](http://rubentd.com/gifplayer/).
+
+
+There was just one problem. Markdown. Again.
+
+Triggering the gifplayer code is as easy as adding a CSS class to an image. Like this:
+
+```html
+<img class="gif" src="http://rubentd.com/img/banana.png">
+```
+
+But native markdown doesn't support adding CSS. Of course, markdown *does* support inline HTML, and we *could have* just written HTML like the line above whenever we wanted to add an animated GIF. But, again, **I'm stubborn**. I like my markdown to be clean. I didn't want to go down a slippery slope of adding special cases for departing from pure markdown.
+
+Now, there are two parameters that you give an image in markdown: the file name, which I needed to tell it where the file is, and alt text, which we often leave blank (I know... accessibility). So I decided to customize the jquery function that initialized gifplayer. Instead of looking for images with a CSS class called 'gif', it would look for images with  alt text set to 'gif'. 
+
+Like this:
+
+```javascript
+<script>
+  $(document).ready(function(){
+    $("img[alt='gif']").gifplayer({ label: 'Play' });
+	});
+</script>
+```
+
+So, now, if we write:
+
+```markdown
+![gif](http://rubentd.com/img/banana.png)
+```
+it'll trigger the gifplayer script!
+
+<img src="http://rubentd.com/img/banana.gif" width=100 alt="party">
+
+You can see it in action on our new article on [common workarounds](https://support.balsamiq.com/tutorials/workarounds/#link-to-alternates). 
+
+(Sorry, it doesn't work on this site. You're gonna have to keep scrolling. ;-))
+
+
+## Challenge #3: The table of contents pages shouldn't be so boring
+
+**TBD:** is it already too long?
```

Comment:
Word count, it's less than 1200 words. But yeah, with all the images, maybe split into two posts?